app/controllers/records_controller.rb

```diff
@@ -1,6 +1,22 @@
 class RecordsController < ApplicationController
   include RecordsControllerBehavior
 
+  def choose_ingest_options
+    @templates = [["Raw (no template)", nil]] + FormTemplate.all.collect {|t| [t.title, t.id]}
+  end
+
+  def new
+    @form = build_form
+    find_template
+    render 'records/new'
+  end
+
+  def edit
+    @form = build_form
```

Comment:
Actually I think I should do that - it would let us remove `new` and `edit` entirely, since those would then be precisely what Hydra Editor already does.